docs/src/api.md

```diff
@@ -22,33 +22,35 @@ GLM.ModResp
 
 The most general approach to fitting a model is with the `fit` function, as in
 ```jldoctest
-julia> using Random
+julia> using GLM, StableRNGs
 
-julia> fit(LinearModel, hcat(ones(10), 1:10), randn(MersenneTwister(12321), 10))
-LinearModel{GLM.LmResp{Vector{Float64}}, GLM.DensePredChol{Float64, LinearAlgebra.CholeskyPivoted{Float64, Matrix{Float64}, Vector{Int64}}}}:
+
+julia> fit(LinearModel, hcat(ones(10), 1:10), randn(StableRNG(12321), 10))
```

Comment:
if we are changing this I would recommend not to use randomly generated data in docs. Maybe e.g. use dataset I from Anscombe's quartet? (then users who know it will visually immediately see that the result is correct)